lib/radioactive_bunnies/deadletter_worker.rb

```diff
@@ -0,0 +1,46 @@
+module RadioactiveBunnies
+  module DeadletterWorker
+    module ClassMethods
+      def register_with_deadletter_worker(worker_class)
+        deadletter_producers << worker_class.name
+      end
+
+      def deadletter_producers
+        @dl_producers ||= []
+      end
+
+      def deadletter_init(worker_opts)
+        return unless !!worker_opts[:deadletter_workers]
+        worker_list = [worker_opts[:deadletter_workers]].flatten
+        exchanges = worker_list.inject([]) do |deadletter_exchange, worker|
+          worker_klass = RadioactiveBunnies::Ext::Util.constantize!(worker)
```

Comment:
Might have missed this in a prior PR. Why is constantize a bang method if its not mutating, but returning a new obj?